src/Examples/UnitModels/Reactors/equilibrium_reactor_testing.ipynb

```diff
@@ -0,0 +1,1135 @@
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Check solver solve status\n",
+    "from pyomo.environ import TerminationCondition\n",
+    "assert results.solver.termination_condition == TerminationCondition.optimal"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Analyze the results of the square problem\n",
+    "\n",
+    "\n",
+    "What is the total operating cost? "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "operating cost = $ 46887895.524803445  per year\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('operating cost = $', value(m.fs.operating_cost), ' per year')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "metadata": {
+    "tags": [
+     "testing",
+     "remove_cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import pytest\n",
+    "assert value(m.fs.operating_cost)/1E6 == pytest.approx(46.887895, abs=1e-3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this operating cost, what conversion did we achieve of methane to hydrogen?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 23,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "====================================================================================\n",
+      "Unit : fs.R101                                                             Time: 0.0\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Unit Performance\n",
+      "\n",
+      "    Variables: \n",
+      "\n",
+      "    Key       : Value      : Fixed : Bounds\n",
+      "    Heat Duty : 2.7836e+07 : False : (None, None)\n",
+      "\n",
+      "------------------------------------------------------------------------------------\n",
+      "    Stream Table\n",
+      "                               Inlet     Outlet  \n",
+      "    Total Molar Flowrate        309.01     429.02\n",
+      "    Total Mole Fraction H2  9.9996e-06    0.50717\n",
+      "    Total Mole Fraction CO  9.9996e-06   0.052286\n",
+      "    Total Mole Fraction H2O    0.75725    0.31800\n",
+      "    Total Mole Fraction CO2 9.9996e-06   0.087587\n",
+      "    Total Mole Fraction CH4    0.24272   0.034965\n",
+      "    Temperature                 500.00     876.72\n",
+      "    Pressure                2.0000e+05 2.0000e+05\n",
+      "====================================================================================\n",
+      "\n",
+      "Conversion achieved =  80.0 %\n"
+     ]
+    }
+   ],
+   "source": [
+    "m.fs.R101.report()\n",
+    "\n",
+    "print()\n",
+    "print('Conversion achieved = ', value(m.fs.R101.conversion)*100, '%')"
```

Comment:
`.format` or f string here too. You can also format to percent using formatting options.